### PR TITLE
Feature/car bump mechanics

### PR DESCRIPTION
**Pull request by DJSteins**
Adds car bump mechanics to the car by using a second box collider around the car that has a bouncy material on it.

### CODE REVIEW

File: Assets/_PartyParkade/Script/Game/Car/CarBounce.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+namespace Assets._PartyParkade.Script.Game.Car
+{
+    class CarBounce : MonoBehaviour
+    {
+        [SerializeField]
+        private ParticleSystem m_sparks = null;
+
+        private void Awake()
+        {
+            m_sparks.gameObject.SetActive(false);
```
Comment (m4rtint) [inline Assets/_PartyParkade/Script/Game/Car/CarBounce.cs:17]: you should set the particle system to not play on awake if you don't want it to show on active

No need to disable the game object.

File: Assets/_PartyParkade/Script/Game/Car/CarBounce.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Game/Car/CarBounce.cs:1]: Missing Header

**Comment by mikeslgoh** [comment]
[Bump sparks](https://trello.com/c/uNTEyWpv/132-bump-sparks)

**Comment by mikeslgoh** [comment]
[Car stops too quickly on collision](https://trello.com/c/sEnOOfV5/425-car-stops-too-quickly-on-collision)

**Pull request merged**